src/components/Footer/Footer.scss

```diff
@@ -0,0 +1,13 @@
+@import '../../common/variables.scss';
+
+.footer {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: $secondary;
+  height: $footerHeight;
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  width: 100%;
```

Comment:
em or %?

Reply:
My idea is the footer to be responsive to the window and always to be 100% of the window width. If I use em the width of footer will be static. If the idea is to be consistent I prefer to change the value of height property to %.